### ISSUE TITLE
address.streetAddress() doesn't use locale.address.street_address template

### ISSUE THREAD

**Issue by erlendfh** [faker-js/faker#88]
### Describe the bug

In fact, most of the `locale.address.*` properties seems to be unused. Am I missing something, is this being used somewhere else, or should I open a PR and propose a fix?

### Reproduction

```javascript
faker.setLocale('nb_NO');
// should print "#{street_name} #{building_number}" - i.e. "Some street 123", not "123 Some street"
console.log(faker.address.streetAddress()); 
```

**Comment by MilosPaunovic** [comment]
PR is definitely welcome.

**Comment by MohdImran001** [comment]
@prisis can I work on it ?

**Comment by Shinigami92** [comment]
@MohdImran001 Let's focus on v6.0.0 first :slightly_smiling_face: 
Otherwise there would be very much to rebase

**Comment by erlendfh** [comment]
Does anyone know the status of this feature? There seems to have been a lot of work put into the translations to support this, but as far as I can tell it's not actually being used at all? Or is there some way of using this "manually" in the _format_ method?

**Comment by pkuczynski** [comment]
I can work on this. Let me know if you want me to...

**Comment by ST-DDT** [comment]
> Or is there some way of using this "manually" in the _format_ method?

In the meantime you can use the formats with the `faker.fake(format: string): string` method.

````ts
faker.fake('#{Address.building_number} #{Address.street_name}');
````

**Comment by ST-DDT** [comment]
I will tackle this, during the implementation I noticed, that this requires the addition of a new method for `buildingNumbers()`, so unfortunately I have to consider this out of scope of v6.1. It will be implemented in v6.2 instead.
